pdf/pdfcanvas.h

```diff
@@ -33,6 +33,7 @@ class PDFCanvas : public QQuickItem
     Q_PROPERTY( QColor linkColor READ linkColor WRITE setLinkColor NOTIFY linkColorChanged )
     Q_PROPERTY( QColor pagePlaceholderColor READ pagePlaceholderColor WRITE setPagePlaceholderColor NOTIFY pagePlaceholderColorChanged )
     Q_PROPERTY( int currentPage READ currentPage NOTIFY currentPageChanged )
+    Q_PROPERTY(QSizeF maxPageSize READ maxPageSize NOTIFY pageSizesReady)
```

Comment:
Signal can be emitted when maxSize didn't change. Small detail, but could make sense to have a separate signal for that.